custom_components/ramses_cc/sensor.py

```diff
@@ -225,42 +218,6 @@ def extra_state_attributes(self) -> dict[str, Any]:
         return attrs
 
 
-class RamsesFaultLog(RamsesDeviceBase):
```

Comment:
Removed as unused. I assume the binary sensor will be the only place this is exposed.

Reply:
This needs sorting, one day.